### DISCUSSION TITLE
Need Help for Backend

### DISCUSSION THREAD

**Discussion by JamshedAlamQaderi** [pedroSG94/RootEncoder#1352] [General]
Hello @pedroSG94 ,

I want to stream from android phone to a backend and from the backend i want to play the stream in the browser. 

Can you tell me how i can approach this?

**Comment by pedroSG94** [comment]
Hello,

First, You need install a server in your backend:
https://github.com/pedroSG94/RootEncoder/wiki/Media-servers-tested

Then you can reproduce the stream in a browser using a HLS player. I think that video tag of html5 support it but you can use this as guide:
https://stackoverflow.com/questions/61933175/playing-hls-file-from-a-server-with-a-html-video-tag
